### PR TITLE
Add BlueWallet.app v6.0.1

### PR DESCRIPTION
**Pull request by maltokyo**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

After making all changes to a cask, verify:

- [x ] The submission is for [a stable version](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#stable-versions) or [documented exception](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#but-there-is-no-stable-version).
- [x ] `brew audit --cask {{cask_file}}` is error-free.
- [ x] `brew style --fix {{cask_file}}` reports no offenses.

Additionally, **if adding a new cask**:

- [x ] Named the cask according to the [token reference](https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md).
- [x ] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues).
- [ x] Checked the cask is submitted to [the correct repo](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask).
- [x ] `brew audit --new-cask {{cask_file}}` worked successfully.
- [x ] `brew install --cask {{cask_file}}` worked successfully.
- [x ] `brew uninstall --cask {{cask_file}}` worked successfully.

### CODE REVIEW

**Comment by miccal** [comment]
Thank you for your contribution to `homebrew-cask` @maltokyo.

Could you please try to add a [`zap` stanza](https://github.com/homebrew/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/zap.md), thanks.

**Comment by maltokyo** [comment]
Added zap, thank you @miccal - please take a look again.

File: Casks/bluewallet.rb [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  sha256 "c76ed6f087c24809c010cb1088b58e8a80fc246f24d4061a5c60051bb0ebb3d6"
+
+  url "https://github.com/BlueWallet/BlueWallet/releases/download/v#{version}/BlueWallet.#{version}.dmg",
+      verified: "github.com/BlueWallet/BlueWallet/"
+  appcast "https://github.com/BlueWallet/BlueWallet/releases.atom"
+  name "BlueWallet"
+  desc "Bitcoin wallet and Lightning wallet"
+  homepage "https://bluewallet.io/"
+
+  zap trash: [
```
Comment (reitermarkus) [inline Casks/bluewallet.rb:12]: This should be after `app`.

Comment (maltokyo) [reply Casks/bluewallet.rb:12]: @reitermarkus thank you - changed, please check now (is it what you meant?)

**Comment by miccal** [comment]
Thank you for your contribution to `homebrew-cask` @maltokyo.

**Review by miccal — APPROVED** [review]

**Comment by maltokyo** [comment]
Hi @miccal  - I am sorry to bother you, but I deleted my local branch, and tried to install anew with the official one.  And now get this, even after restart and `brew update`.  I am not sure if there is some lead time needed, or if I did something wrong locally?  Please help if you have time.

$ brew install bluewallet
```
==> Searching for similarly named formulae...
Error: No similarly named formulae found.
Error: No available formula or cask with the name "bluewallet".
==> Searching for a previously deleted formula (in the last month)...
Error: No previously deleted formula found.
==> Searching taps on GitHub...
Error: No formulae found in taps.
```

**Comment by miccal** [comment]
@maltokyo I am not sure why -- it is definitely available to me, and our analytics even show it has been installed a few times:
```
|-> brew info bluewallet
bluewallet: 6.0.1
https://bluewallet.io/
Not installed
From: https://github.com/Homebrew/homebrew-cask/blob/HEAD/Casks/bluewallet.rb
==> Name
BlueWallet
==> Description
Bitcoin wallet and Lightning wallet
==> Artifacts
BlueWallet.app (App)
==> Analytics
install: 4 (30 days), 4 (90 days), 4 (365 days)
```

**Comment by maltokyo** [comment]
Thank you, resolved in the closed issue here: https://github.com/Homebrew/homebrew-cask/issues/96810

**Pull request merged**